bootstrap/bukkit/src/main/java/org/geysermc/platform/bukkit/GeyserBukkitConfiguration.java

```diff
@@ -151,6 +151,11 @@ public boolean isCacheChunks() {
         return true; // We override this as with Bukkit, we have direct access to the server implementation
     }
 
+    @Override
+    public boolean isAllowCustomSkulls() {
+        return config.getBoolean("allow-custom-skulls", false);
```

Comment:
Should we leave in the config option, these should all be set to true like the config.